cunumeric/module.py

```diff
@@ -2371,8 +2371,8 @@ def choose(a, choices, out=None, mode="raise"):
     return a.choose(choices=choices, out=out, mode=mode)
 
 
-@add_boilerplate("c", "a")
-def compress(c, a, axis=None, out=None):
+@add_boilerplate("condition", "a")
+def compress(condition, a, axis=None, out=None):
```

Comment:
Note: I also changed the parameter name here to match numpy, otherwise numpy code that passed the param as a kwarg would break when switched to `cunumeric`.

ref: https://numpy.org/doc/1.22/reference/generated/numpy.compress.html